back/src/adapters/primary/config/createUseCases.ts

```diff
@@ -386,6 +387,12 @@ export const createUseCases = (
           uowPerformer,
           saveNotificationAndRelatedEvent,
         ),
+      notifyBeneficiaryAndEnterpriseThatConventionIsCancelled:
+        new NotifyBeneficiaryAndEnterpriseThatConventionIsCancelled(
+          uowPerformer,
+          saveNotificationAndRelatedEvent,
+        ),
+
       notifyAllActorsThatConventionIsDeprecated:
```

Comment:
je serai tenté de renommer ca aussi :
```suggestion
      notifyConventionDeprecated:
```